test/jdk/javax/swing/plaf/basic/BasicTextUI/8001470/bug8001470.java

```diff
@@ -48,21 +53,27 @@ public void run() {
                 JPanel container = (JPanel) frame.getContentPane();
                 container.setLayout(new GridLayout(2,1));
 
-                textField1 = new JTextField("\u0e01");
-                textField2 = new JTextField("\u0c01");
+
+                textField1 = new JTextField("\u2588");
+                textField2 = new JTextField("\u2588");
```

Comment:
You do not need to test the same character twice

Reply:
This isn't right. The test MUST use the Indic and Thai characters else the test is pointless.
The problem of them being unavailable in the font should be solved by using Font.canDisplay(char)
to find a font that DOES support them.
If no font supports them, then it becomes a question of policy .. do you want the test to silently pass
or should the test fail because the system running this test really should have those fonts.

Reply:
Updated with font canDisplay check